filer/private/sass/settings/_custom.scss

```diff
@@ -47,6 +48,8 @@ $screen-mobile: 320px;
 $screen-tablet: 720px;
 $screen-desktop: 975px;
 
+$screen-filer: 810px;
```

Comment:
i was just thinking if we could come up with something that is a bit more descriptive than just `filer`